_posts/2024-01-19-yannick-horrix-opentofu.md

```diff
@@ -45,54 +49,73 @@ The upcoming version 1.7 aims to introduce more community requested features not
-Terraform configuration, on the other hand, can be more verbose and complex to maintain.
-## Licensing and Pricing
+
+OpenTofu and Terraform both use Hashicorp Configuration Language (HCL) to define their resources. HCL is declarative, describing an intended goal rather than the steps to reach that goal.
+
+In terms of configuration options, both are also similar. 
+
+## Licensing
+
 Both Terraform and OpenTofu are open-source tools, providing free usage.
 
-As said, OpenTofu uses the Mozilla Public License (MPL), which allows developers to freely use, modify, and distribute software for both commercial and non-commercial use.
+As said, OpenTofu uses the Mozilla Public License (MPL 2.0), which allows developers to freely use, modify, and distribute software for both commercial and non-commercial use.
 
-Terraform has now shifted to the Business Source License (BUSL). This also permits free use of source code, including commercial use, except when it’s used to provide an offering that competes with HashiCorp.
+Terraform has now shifted to the Business Source License (BUSL 1.1). This also permits free use of source code, including commercial use, except when it’s used to provide an offering that competes with HashiCorp.
 An example of this could be running Terraform in a hosted way in CI/CD and offering this as a production service.
 
 So while, Terraform remains free to use in your projects, there can be an issue when incorporating Terraform in a service offering.
 
-In terms of paid offerings, Terraform does offer multiple options with Terraform Cloud, a platform managing Terraform code, availaible as a hosted service. 
-Additionally, organizations with advanced security and compliance needs can purchase Terraform Enterprise. 
-This is a self-hosted distribution of Terraform Cloud which offers a private instance that includes the advanced features available in Terraform Cloud.
+
+
+## Pricing and Offerings
+
+In terms of pricing, both tools are free to use, with some limitations for commercial use when it comes to Terraform.
+
+Hashicorp does offer paid solutions, starting with Terraform Cloud, a centralized platform for managing Terraform code. Key features include version control integration, secure variable storage, remote state management, and policy enforcement, enabling organizations to efficiently maintain control over their cloud infrastructure.
+
+[Pricing](https://www.hashicorp.com/products/terraform/pricing?product_intent=terraform){:target="_blank" rel="noopener noreferrer"} for Terraform Cloud follows a Resources Under Management (RUM) model, where Terraform counts the amount of objects it manages and calculates cost accordingly. 
+The Standard Tier offering is billed at $0.00014 per hour per resource. For the Plus and Enterprise tiers, pricing is negotiated directly with Hashicorp sales.
+
+Let's take the example os Terraform Enterprise. This is a self-hosted solution offered as a private installation rather than a SaaS solution.
```

Comment:
```suggestion
Let's take the example for Terraform Enterprise. 
This is a self-hosted solution offered as a private installation rather than a SaaS solution.
```